server.js

```diff
@@ -123,6 +123,18 @@ module.exports.initExpress = function () {
     base: 10,
     round: 0,
   });
+  app.post(
+    '/pl/scan_paper',
```

Comment:
I really think we should have authorization here - without that, any old student would be able to upload arbitrary PDFs.

Reply:
Done in Part 2.